os_installer2/mainwindow.py

```diff
@@ -167,7 +167,7 @@ def __init__(self, app):
         self.set_default_size(768, 500)
 
         # Main "install" page
-        self.installer_page = Gtk.VBox(0)
+        self.installer_page = Gtk.Box(Gtk.Orientation.VERTICAL, 0)
```

Comment:
pygi can be buggy, probably safer to explicitly use `Gtk.Box.new(Gtk.Orientation.VERTICAL, 0)`